src/lib/ObjectStore.ts

```diff
@@ -41,13 +41,17 @@ class ObjectStore {
     }
 
     // http://w3c.github.io/IndexedDB/#retrieve-multiple-keys-from-an-object-store
-    public getAllKeys(range: FDBKeyRange, count?: number) {
+    public getAllKeys(
+        range: FDBKeyRange,
+        count?: number,
+        direction?: "next" | "prev",
+    ) {
         if (count === undefined || count === 0) {
             count = Infinity;
         }
 
         const records = [];
-        for (const record of this.records.values(range)) {
+        for (const record of this.records.values(range, direction ?? "next")) {
```

Comment:
Default direction is "next" so ` ?? "next"` can be deleted in all of these, I think